rclcpp_action/test/test_client.cpp

```diff
@@ -1035,3 +1035,20 @@ TEST_F(TestClientAgainstServer, execute_rcl_errors)
       rclcpp::exceptions::RCLError);
   }
 }
+
+TEST_F(TestClientAgainstServer, test_configure_introspection)
+{
+  auto action_client = rclcpp_action::create_client<ActionType>(client_node, action_name);
+
+  EXPECT_THROW(
+    action_client->configure_introspection(
+      nullptr, rclcpp::SystemDefaultsQoS(), RCL_SERVICE_INTROSPECTION_CONTENTS),
+      std::invalid_argument);
+
+  EXPECT_NO_THROW(
+    action_client->configure_introspection(
+      client_node->get_clock(), rclcpp::SystemDefaultsQoS(), RCL_SERVICE_INTROSPECTION_CONTENTS));
```

Comment:
corresponding publishers for service events are verified in rcl test cases, so i think that it is fine just to make sure this API passes here.

Reply:
Yes.